### ISSUE TITLE
Looking for help / maintainers

### ISSUE THREAD

**Issue by yanns** [sangria-graphql/sangria#1084]
In 2019 I took over the maintenance of sangria from [Oleg](https://github.com/sangria-graphql/sangria/issues/445) with some [others](https://github.com/sangria-graphql/sangria/issues/446).

In the past months, I've spent less time on this project.

I'm also looking for help and other maintainers to step in.
I can provide help and guidance.
I can stay in the maintenance team if desired even though I'll not be very active on the long term.

**Comment by avelx** [comment]
Yeah, would be interested. I think contributing is the best way to learn; so happy to try and see how it goes over time

**Comment by yanns** [comment]
Thanks a lot @avelx and @spiliopoulos .
I've added you to the contributors team.
You could start by reviewing PRs, included those to update dependencies.
If you have ideas on where to contribute, don't hesitate to suggest.
